cross.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM dockercore/golang-cross:1.12.15
+FROM dockercore/golang-cross:1.12.17
```

Comment:
Perhaps update this to go 1.13.x (there's no image yet for Go 1.14, but latest versions are on 1.13.x). Waiting for https://github.com/docker/golang-cross/pull/58 to be merged, after which I'll tag go 1.13.11 for the image

Reply:
Changed it in PR #1560 and rebased this PR on top of that.